stable/spark/README.md

```diff
@@ -0,0 +1,68 @@
+# Apache Spark Helm Chart
+
+Apache Spark is a fast and general-purpose cluster computing system.
+
+* http://spark.apache.org/
+
+## Chart Details
+This chart will do the following:
+
+* 1 x Spark Master with port 8080 exposed with a nodePort service (webUi)
```

Comment:
nodeport cannot be less than 30000 usually, are you sure that you are exposing on 8080?